src/include/expression/aggregate_expression.h

```diff
@@ -116,10 +118,31 @@ class AggregateExpression : public AbstractExpression {
     }
   }
 
-  virtual void Accept(SqlNodeVisitor* v) override { v->Visit(this); }
+  virtual void Accept(SqlNodeVisitor *v) override { v->Visit(this); }
+
+  const std::string GetInfo(int num_indent) const override {
```

Comment:
I thought that we were moving these into the .cpp files? And I thought that we were making the `GetInfo(int num_indent)` methods protected?

Reply:
I thought the change from all public to protected or private will be the next phase?